react-common/components/share/ShareInfo.tsx

```diff
@@ -323,7 +324,7 @@ export const ShareInfo = (props: ShareInfoProps) => {
             {showSimulator && shareState !== "gifrecord" &&
                 <div className="project-share-thumbnail">
                     {thumbnailUri
-                        ? <img src={thumbnailUri} />
+                        ? <img src={thumbnailUri} alt={lf("Preview of your code running on a micro:bit")} aria-label={lf("Micro:bit simulator preview")}/>
```

Comment:
This alt-text is very micro:bit specific. You can use the boardname property of the apptarget to get the name of the hardware for the target. Or just use the more generic term "simulator" without mentioning the board at all.